Dockerfile

```diff
@@ -8,13 +8,12 @@ WORKDIR /app
 # Install dependencies first (assuming they'll change less frequently, this
 # will make image re-builds faster)
 ADD package.json package.json
-RUN npm install
+RUN yarn
```

Comment:
The only advantage I see to doing this is that we have the lockfile. Is there any other reason to use yarn here rather than npm? (Better caching doesn't apply, since Docker will throw that out.)